portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,42 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funfacts =
+      ['I love hiking, biking, and spending time outdoors.', 
+      'At Columbia, I am part of the Pops Orchestra, an ensemble dedicated to arranging and playing pop pieces for orchestra.',
+       'I have two pet cats.',
+        'My favorite movie is The Matrix.', 
+        'I am a middle child - I have an older brother and two younger sisters.', 
+        'My favorite food is garbanzo beans.', 
+        'Currently, I am attempting to grow potatoes outside.',
+        'My favorite animal is a panda.'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const funfact = funfacts[Math.floor(Math.random() * funfacts.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  // Add it to the page in a talbe.
+  var funFactTable = document.getElementById('funfact-table');
+  const funFactRowText = '<tr><td>' + funfact + '</td></tr>';
+
+  if(!tableText.includes(funFactRowText)){
```

Comment:
Instead of checking the string, are u able to create 2 list
1) var displayedFunFact
2) var unusedFunFact

Then, you don't need all the if/else if/else condition

When user want a random Greeting
1) Check if len(unusedFunFact) == 0 --> return that all entries are used
2) get a fun fact using unusedFunFact --> can avoid getting repeat values
3) After you get a fun fact, remove the fun fact from unusedFunFact and add it to displayedFunFact.

There are different way to generate the fun fact, but just giving a rough idea.

Overall very creative way to generate fun facts! (was also very cool to play around with the website)